### DISCUSSION TITLE
Adding multiple classes to elements (mv-attr-* or mv-action?)

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#848] [Q&A] [answered]
Hi

Is it possible to use mavo not only to set a class to an element,

but also to add/delete new classes to the ones that are already set on elements?

I've been trying to do it in this [codepen](https://codepen.io/galinhalx/pen/BaYLJOJ?editors=1010)
but I've been only able to set it with value I'm passing with `mv-action`

Anything available with `mv-attr-*` maybe? (Would be an intesting feature that would replace a lot of extra javascript to manage classes)

**Comment by LeaVerou** [comment, marked as answer]
I think you are trying to use `and` to join two text values. `and` is for conjunctions, to concatenate text values please use `&`.

**Reply by GalinhaLX** [reply]
Thanks Lea, works perfect
